### ISSUE TITLE
SDG 5: Rename and replace image with correct color one

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#4255]
### Dependency
- #4183 

### Overview
We need to replace the Sustainable Development Goals (SDG) images in our repository so that the SDG svgs are the correct color and follow the [United Nations' SDG guidelines](https://www.un.org/sustainabledevelopment/wp-content/uploads/2019/01/SDG_Guidelines_AUG_2019_Final.pdf). 

### Details
SDG is an acronym for the [United Nations Development Programme Sustainable Development Goals](https://www.undp.org/sustainable-development-goals).

### Action Items
- [x] In the `assets/images/sdg` directory, delete `gender-equality.svg`
- [x] From the HackforLA.org website drive's [HackforLA.org website/SDG/SDG images](https://drive.google.com/drive/folders/1kJdguLXnIRv13IRh_4v4lrYDG7K4FbML) folder, download the following svg: `E-WEB-Goal-05.svg`
- [x] Add the svg you just downloaded into the `assets/images/sdg` directory
- [x] Rename that svg so that it becomes `sdg5.svg`
- [x] In `_includes/about-page/about-card-sustainability.html`, update the path to the SVG you just added by changing
  ```
  <img src="/assets/images/sdg/gender-equality.svg" alt="5 Gender Equality" />
  ```
  to
  ```
  <img src="/assets/images/sdg/sdg5.svg" alt="5 Gender Equality" />
  ```
- [x] Using Docker, check the About page's Sustainability section (https://www.hackforla.org/about/#sustainability) to check that the SDG image shows up on the same spot as on the live webpage
  - Note: The size of the image may look different compared to the website. This will be addressed in issue #4229. 

#### Merge Team, Technical Lead, or Product Manager
- [ ] Once the pull request associated with this issue is merged, go to each of the following issues and check if all the issues listed under the `Dependency` section are closed. If the dependencies are all closed, move those issues into the `New Issue Approval` column, remove the `Dependency` label on it, and add a `ready for dev lead` label to it.
  - [x] #4229

### Resources/Instructions
- [assets/images/sdg](https://github.com/hackforla/website/tree/gh-pages/assets/images/sdg) directory
- [HackforLA.org website/SDG/SDG images](https://drive.google.com/drive/folders/1kJdguLXnIRv13IRh_4v4lrYDG7K4FbML)
- [_includes/about-page/about-card-sustainability.html](https://github.com/hackforla/website/blob/gh-pages/_includes/about-page/about-card-sustainability.html)
- About page's Sustainability section: https://www.hackforla.org/about/#sustainability
- This issue is tracked in #4153
#### United Nations' SDG Information
- [United Nations' SDG guidelines](https://www.un.org/sustainabledevelopment/wp-content/uploads/2019/01/SDG_Guidelines_AUG_2019_Final.pdf)
- [United Nations' SDG Communications materials](https://www.un.org/sustainabledevelopment/news/communications-material/)
- [United Nations Development Programme Sustainable Development Goals](https://www.undp.org/sustainable-development-goals)

**Comment by JessicaLucindaCheng** [comment]
Once this issue is in the New Issue Approval column, a dev lead needs to review the issue and determine if it is ready for a PM to prioritize it. If it isn't ready, a dev lead will need to edit this issue so that it is ready for a PM to prioritize it.

**Comment by thedvo** [comment]
Availability: M-F 3pm-7pm PST
ETA: April 11 EOD
